packages/fiori/test/pages/UserMenu.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>User Menu</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<script data-ui5-config type="application/json">
+		{
+			"rtl": false
+		}
+
+	</script>
+
+	<script src="%VITE_BUNDLE_PATH%" type="module"></script>
+	<style>
+		#btnOpenUserMenu {
+			float: right;
+		}
+	</style>
+</head>
+
+<body>
+
+<ui5-shellbar id="shellbar" primary-title="Corporate Portal">
+	<ui5-avatar slot="profile">
+		<img src="./img/man_avatar_1.png"/>
+	</ui5-avatar>
+	<img slot="logo" src="./img/sap-logo-svg.svg" />
+</ui5-shellbar>
+<ui5-user-menu id="userMenuShellBar" show-manage-account show-other-accounts show-add-account>
+	<ui5-user-menu-account slot="accounts"
+						   avatar="./img/man_avatar_1.png"
```

Comment:
I am not sure how it will look for the other parts of the code (in the UserMenu), but worth looking into having the avatar as slot to be more flexible, as shapes sizes can be requested in future and we will need to add properties like avatarSize, avatarShape. With the slot, the apps will be able to configure them.

Reply:
I will definitely try. It makes a lot of sense.

Reply:
We discussed that idea and agreed to leave it as it now because:
Avatar size is specified in the design both for selected user and other users to be exact size (L and S respectively)
which makes it impossible to be defined as one slot.